pkg/api/v1beta3/dynakube/extensions.go

```diff
@@ -10,14 +10,6 @@ type ExtensionsSpec struct {
 	Prometheus PrometheusSpec `json:"prometheus,omitempty"`
 }
 
-type TemplatesSpec struct {
```

Comment:
Why is this removed?

Reply:
see the comment from @aorcholski 

https://github.com/Dynatrace/dynatrace-operator/pull/3595#discussion_r1709124154

it was moved to dynakube_types.go